packages/beacon-node/src/chain/interface.ts

```diff
@@ -199,3 +202,7 @@ export type SSZObjectType =
   | "signedAggregatedAndProof"
   | "syncCommittee"
   | "contributionAndProof";
+
+export type CommonBlockBody = phase0.BeaconBlockBody &
+  Pick<capella.BeaconBlockBody, "blsToExecutionChanges"> &
+  Pick<altair.BeaconBlockBody, "syncAggregate">;
```

Comment:
`Omit<allForks.BeaconBlockBody, "executionPayload">` is not type-safe, so why used explicit attributes here.